content/user/security/relocate_download_folder.md

```diff
@@ -27,13 +27,28 @@ To do this, you must:
     - `/includes/configure.php`
     - `/YOURADMIN/includes/configure.php`
 
-You'll need to edit the line that says this:  
+In Zen Cart 1.5.5 and higher: 
+
+There is no line setting `DIR_FS_DOWNLOAD` in these versions of Zen Cart.  You must *add* a line to the bottom of your configure.php files that says something like 
+
+> `define('DIR_FS_DOWNLOAD', '/home/my_user_name/download/');`
+
+In Zen Cart 1.5.4 and lower: 
+
+> You'll need to edit the line that says this:  
   
-`define('DIR_FS_DOWNLOAD', DIR_FS_CATALOG . 'download/');`
+> `define('DIR_FS_DOWNLOAD', DIR_FS_CATALOG . 'download/');`
   
-to look something like this:  
+> to look something like this:  
   
-`define('DIR_FS_DOWNLOAD', '/home/my_user_name/download/');`
+> `define('DIR_FS_DOWNLOAD', '/home/my_user_name/download/');`
   
 Specifically, you need to indicate the exact complete path for the webserver's filesystem that points to your download folder.  You could use your `DIR_FS_CATALOG` entry in your configure.php file as a reference, and make appropriate adjustments.  Your FTP program and/or your webserver's hosting control panel should be able to help you understand the complete path.
 
+As an example, if your `DIR_FS_CATALOG` value is set as follows: 
+
+`define('DIR_FS_CATALOG', '/home/jimmy/public_html/store/');`
+
+then you would want 
+
+`define('DIR_FS_CATALOG', '/home/jimmy/download/');`
```

Comment:
Shouldn't this last line refer to DIR_FS_DOWNLOAD and not DIR_FS_CATALOG?

Reply:
Ugh yes.